docs/ocean/design_docs/vert-lagrang-remap/vert-lagrang-remap.rst

```diff
@@ -0,0 +1,302 @@
+
+Vertical Lagrangian-remap method
+================================
+
+date: 2020/11/24
+
+Contributors: Darren Engwirda, Xylar Asay-Davis, Carolyn Begeman
+
+
+
+Summary
+-------
+
+The vertical Lagrangian-remap method is to be implemented to support the choice 
+of hybrid vertical coordinates. It can be seen as an extension of the Arbitrary 
+Lagrangian-Eulerian method which is already implemented in MPAS-Ocean, a key 
+difference being that the solution of diasurface transport is coordinate-free. 
+This has the advantage of removing the vertical CFL constraint, which is of 
+particular concern for an evolving ice shelf geometry in terrain-following 
+coordinate schemes. Generalization of the vertical coordinate in the horizontal 
+equations of motion is not in the scope of this development. This development 
+will include some infrastructure for specifying the target grid for regridding. 
+The minimum criteria for success are not degrading the accuracy of the solution 
+or the performance by more than [an acceptable threshold] for several basic test 
+cases... [hmm, not quite sure how to write the success statement if not "meets requirements"].
+
+
+Requirements
+------------
+
+Requirement: remapping
+^^^^^^^^^^^^^^^^^^^^^^
+
+Date last modified: 2020/12/04
+
+Contributors: Darren Engwirda, Xylar Asay-Davis, Carolyn Begeman
+
+Requirement: A grid that relaxes to an analytical expression for a target grid with non-negative layer thicknesses.
+
+Requirement: hybrid coordinates
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Date last modified: 2020/12/04
+
+Contributors: Darren Engwirda, Xylar Asay-Davis, Carolyn Begeman
+
+Requirement: The ability to implement hybrid coordinate systems.
```

Comment:
I'm not 100% clear on what hybrid coordinates means in this context.  Could this be fleshed out?  Is this maybe a requirement for a future design document?

Reply:
Per the discusion in our 12/8 meeting, hybrid coordinates are left for a future design doc